### PR TITLE
Initial pr

### PR DESCRIPTION
**Pull request by HarithaIBM**
Pinentry - initial commits

### CODE REVIEW

File: buildenv [hunk 1]
```diff
@@ -0,0 +1,37 @@
+export ZOPEN_TARBALL_URL="https://www.gnupg.org/ftp/gcrypt/pinentry/pinentry-1.2.1.tar.bz2"
+export ZOPEN_TARBALL_DEPS="make bzip2 tar curl perl automake autoconf m4 sed gettext zoslib diffutils libassuan libgpgerror"
+export ZOPEN_TYPE="TARBALL"
+export ZOPEN_COMP=CLANG
+export ZOPEN_CONFIGURE_OPTS=" --prefix=/home/haritha/zopen/prod/pinentry-1.2.1/ --enable-pinentry-tty"
+
+PINENTRY_VERSION="1.2.1"
```
Comment (IgorTodorovskiIBM) [inline buildenv]: Suggest moving this up, and using it in the tarball url so that you only have to change one place.

File: buildenv [hunk 2]
```diff
@@ -0,0 +1,37 @@
+export ZOPEN_TARBALL_URL="https://www.gnupg.org/ftp/gcrypt/pinentry/pinentry-1.2.1.tar.bz2"
+export ZOPEN_TARBALL_DEPS="make bzip2 tar curl perl automake autoconf m4 sed gettext zoslib diffutils libassuan libgpgerror"
+export ZOPEN_TYPE="TARBALL"
+export ZOPEN_COMP=CLANG
+export ZOPEN_CONFIGURE_OPTS=" --prefix=/home/haritha/zopen/prod/pinentry-1.2.1/ --enable-pinentry-tty"
```
Comment (IgorTodorovskiIBM) [inline buildenv]: Use ZOPEN_EXTRA_CONFIGURE_OPTS, that way you don't need to specify prefix.

**Review by MikeFultonDev — APPROVED** [review]
approve pending igor's suggestions

File: buildenv (hunk 2 above)
Comment (HarithaIBM) [reply buildenv]: Yes, it helped, I have modified it.

File: buildenv (hunk 1 above)
Comment (HarithaIBM) [reply buildenv]: Right, i have done it, Igor! Thank you!

> approve pending igor's suggestions

Sure, Thanks Mike!

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM

**Comment by HarithaIBM** [comment]
zopen_check_results processing is yet to be done, tracked [here](https://github.com/ZOSOpenTools/pinentryport/issues/2)

**Pull request merged**